src/network/mcpe/handler/InGamePacketHandler.php

```diff
@@ -869,6 +872,21 @@ public function handleLabTable(LabTablePacket $packet) : bool{
 		return false; //TODO
 	}
 
+	public function handleLecternUpdate(LecternUpdatePacket $packet) : bool{
+		//TODO: Cancel page turn event
+		//Drop book is handled on server side with an interact event.
+		//Total pages is gathered from book.
```

Comment:
You should check that the total pages match. If it doesn't, it means there's a desync between client/server (maybe the book was changed and the client didn't know about it before it sent this packet). In such case, the action should be ignored.